packages/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -163,13 +164,14 @@ public NetworkingModule(ReactApplicationContext context, String defaultUserAgent
     this(context, defaultUserAgent, OkHttpClientProvider.createClient(context), null);
   }
 
-  public static void setCustomClientBuilder(CustomClientBuilder ccb) {
+  public static void setCustomClientBuilder(com.facebook.react.modules.network.CustomClientBuilder ccb) {
     customClientBuilder = ccb;
   }
 
-  public static interface CustomClientBuilder {
-    public void apply(OkHttpClient.Builder builder);
-  }
+  /**
+   * @deprecated To be removed in a future release. See https://github.com/facebook/react-native/pull/37798#pullrequestreview-1518338914
+   */
+  public static interface CustomClientBuilder extends com.facebook.react.modules.network.CustomClientBuilder {}
```

Comment:
Can we also `@Deprecate` this method?

Reply:
Sure! Just added that, CICD re-running now.